backend/src/services/common/MeanTimeToRecovery.ts

```diff
@@ -0,0 +1,50 @@
+import { DeployInfo, DeployTimes } from "../../models/pipeline/DeployTimes";
+import {
+  AvgMeanTimeToRecovery,
+  MeanTimeToRecovery,
+  MeanTimeToRecoveryOfPipeline,
+} from "../../contract/GenerateReporter/GenerateReporterResponse";
+
+export function calculateMeanTimeToRecovery(
```

Comment:
This function is a little bit complicated. Could you extract independent functionality to another one? 
It ensure that each function has a single responsibility, high readability and easy for unit test.

Reply:
hi @lvj0525, I've refactored that method. Plz review when you have time.